src/fin_webapp/Gemfile

```diff
@@ -0,0 +1,53 @@
+source "https://rubygems.org"
+
+ruby "3.3.0"
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem "rails", "~> 7.1.3"
+
+# Use sqlite3 as the database for Active Record
+gem "sqlite3", "~> 1.4"
```

Comment:
remove this gem as its not required and you dont use any database

Reply:
parking for laters as require rails/all is not working as intented and might need more time to figure whats happening underneath the hood in Rails 7.